vochain/genesis/genesis.go

```diff
@@ -118,8 +104,8 @@ var devGenesis = GenesisDoc{
 }
 
 var stageGenesis = GenesisDoc{
-	GenesisTime: time.Date(2023, time.March, 10, 10, 0, 0, 0, time.UTC),
-	ChainID:     "vocdoni-stage-3",
+	GenesisTime: time.Date(2023, time.April, 10, 10, 0, 0, 0, time.UTC),
```

Comment:
Please next time make it clear that DEV & STG envs will be affected as these are breaking consensus changes.

Reply:
All commits will be squashed. The final message will contain the breaking notice.